tools/picard.js

```diff
@@ -290,14 +299,16 @@ function build_ui () {
         default_width: RESOLUTIONS[2][0],
         default_height: RESOLUTIONS[2][1],
     });
+    widgets.reset_theme_box = new ResetThemeBox();
```

Comment:
Since you don't need the reset-theme-box anywhere else, you can just keep it local to this function.